activerecord/test/cases/persistence_test.rb

```diff
@@ -168,6 +169,16 @@ def test_becomes_includes_changed_attributes
     assert_equal %w{name}, client.changed
   end
 
+  def test_becomes_includes_association_cache
+    assert Company.count > 0, "company should have preloaded fixtures"
```

Comment:
It feels like we shouldn't need this. Could the line below this one be changed to `company = Company.preload(:contracts).first`, allowing us to remove this line?